docs/build-intercept.rst

```diff
@@ -62,41 +66,45 @@ your target system, it can be omitted during UnifyFS configuration by using
 the ``--without-hdf5`` configure option.
 
 ---------------------------
-How to Build UnifyFS
+
+---------------------------
+Building UnifyFS with Spack
 ---------------------------
 
+Full Build
+**********
+
 To install all dependencies and set up your build environment, we recommend
 using the `Spack package manager <https://github.com/spack/spack>`_. If you
 already have Spack, make sure you have the latest release or if using a clone
 of their develop branch, ensure you have pulled the latest changes.
 
-Building with Spack
-********************
-
-These instructions assume that you do not already have a module system installed
-such as LMod or Environment Modules. If your system already has
-LMod installed then installing the environment-modules package with Spack
-is unnecessary (so you can safely skip that step).
+.. _build-label:
 
-First, install Spack if you don't already have it:
+Install Spack
+^^^^^^^^^^^^^
 
 .. code-block:: Bash
 
     $ git clone https://github.com/spack/spack
-    $ ./spack/bin/spack install environment-modules
+    $ # optionally create a packages.yaml specific to your machine
     $ . spack/share/spack/setup-env.sh
 
 Make use of Spack's `shell support <https://spack.readthedocs.io/en/latest/getting_started.html#add-spack-to-the-shell>`_
 to automatically add Spack to your ``PATH`` and allow the use of the ``spack``
 command.
 
-Then install UnifyFS:
+Install UnifyFS
+^^^^^^^^^^^^^^^
 
 .. code-block:: Bash
 
     $ spack install unifyfs
     $ spack load unifyfs
 
+If the most recent changes on the develop branch of UnifyFS are desired, then do
```

Comment:
rather than "develop branch", how about "development branch ('dev')"

Reply:
Good call. I mixed up which project calls the branch `dev` and which calls it `develop`.